src/components/Section/Section.scss

```diff
@@ -0,0 +1,56 @@
+@import '~theme/base/variables';
+
+
+.section {
+    background-color: transparent;
+}
+
+.inner {
+    margin: 0 auto;
+    
+    @media screen and (min-width: 767px) {
+        width: 90%;
+        max-width: 800px;    
+    }
+
+    @media screen and (min-width: 1024px) {
+        width: 90%;
+        max-width: 900px;
+    }
+
+    @media screen and (min-width: 1400px) {
+        width: 90%;
+        max-width: 1200px;
+    }
+}
+
+
+.hasPaddingHorizontal {
+    padding-left: 30px; 
+    padding-right: 30px;
+}
+
+.hasPaddingVertical {
+    padding-top: 45px;
```

Comment:
This can be shorthanded to padding: 45px 0px;

Reply:
Well, not really. I don't want to edit the horizontal padding property value. Doing this we are able to do the 4 posibilities:
- None
- Only vertical
- Only horizontal
- Both

Reply:
Okay... but if you only want to edit the vertical, padding: 45px 0px is exactly the same as padding-top: 45px and padding-bottom: 45px. It's just more succinct.

Reply:
No it's not the same! If I am wrong please make me understand it hahah. 
 says explicitly that I want `45px` on **top** and **bottom** but **says nothing about the left and right**.
```
padding-top: 45px;
padding-bottom:45px
```
Instead, if I say
```padding: 45px 0;```
I am saying **I want 45px on top and bottom**.

If we do: 
```
.hasVerticalPadding {
    padding: 45px 0;
}

.hasHorizontalPadding {
    padding: 0 45px;
}
```
And now I add both classes to an element, because of the CSS Cascade **(I don't know in `css modules`)**. Our element will have `padding: 45px 0` because is the first style computed. We will not be able either to combine them so we are losing 2 possibilities to combine.